aodntools/timeseries_products/velocity_hourly_timeseries.py

```diff
@@ -41,14 +39,14 @@ def cell_velocity_resample(df, binning_function, is_WCUR):
     return UCUR, VCUR, WCUR, DEPTH
 
 
-def get_resampled_values(nc_cell, ds, slice_start, varlist, binning_fun, epoch, one_day, is_WCUR):
+def get_resampled_values(nc_cell, ds, slice_start, varlist, binning_function, epoch, one_day, is_WCUR):
     """
```

Comment:
too many arguments

Reply:
Agree. I have removed `varlist` and the last three.